### PR TITLE
Update for review (esri/intl)

### PR DESCRIPTION
**Pull request by kellyhutchins**
Updated to use esri/intl methods so we could review and discuss best approach for setting the locale and RTL. Note we can't merge this until next release is live but wanted to get it in for discussion now.

### CODE REVIEW

File: ApplicationBase.ts [hunk 1]
```diff
@@ -396,7 +393,7 @@ class ApplicationBase {
 
   private _mixinSettingsDefaults(settings: ApplicationBaseSettings): void {
     const userEnvironmentSettings = settings.environment;
-    const userLocalStorageSettings = settings.localStorage;
+    // const userLocalStorageSettings = settings.localStorage;
```
Comment (driskull) [inline ApplicationBase.ts]: Can we remove this?

File: ApplicationBase.ts [hunk 2]
```diff
@@ -408,10 +405,10 @@ class ApplicationBase {
       ...userEnvironmentSettings
     };
 
-    settings.localStorage = {
+    /*settings.localStorage = {
```
Comment (driskull) [inline ApplicationBase.ts]: this too?

File: ApplicationBase.ts [hunk 3]
```diff
@@ -532,24 +529,13 @@ class ApplicationBase {
     return (await portal.queryGroups(params)) as __esri.PortalQueryResult;
   }
 
-  private _loadItem(id: string): Promise<PortalItem> {
+  private _loadItem(id: string): IPromise<any> {
```
Comment (driskull) [inline ApplicationBase.ts]: can we type this instead of having any?

**Review by driskull — APPROVED** [review]
This looks good @kellyhutchins! just a few cleanup suggestions.

**Review by rslibed — APPROVED** [review]
👍 Looks good!

**Comment by kellyhutchins** [comment]
Thanks I will update with Matt's suggestions and merge.

File: ApplicationBase.ts (hunk 1 above)
Comment (kellyhutchins) [reply ApplicationBase.ts]: yep

File: ApplicationBase.ts (hunk 3 above)
Comment (kellyhutchins) [reply ApplicationBase.ts]: Yeah I wonder why I switched to any - I'll look at it it could have been an old update that I accidentally merged back in

**Comment by kellyhutchins** [comment]
Ok updates made. I'll merge when next release goes out and update npm

**Pull request merged**